vm-setup.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+# This script is a customization Script and it is used to set instances launched using openstack. 
+
+# update things.
+sudo su
+apt-get -y update && apt-get upgrade -y
+
+# install maven, pidstat and htop. 
+apt install maven -y
+apt install sysstat -y
+apt install htop -y
+
+# install wrk2.
+apt-get install build-essential libssl-dev git -y
+git clone https://github.com/giltene/wrk2.git
+cd wrk2
+make
+cp wrk /usr/local/bin
+cd ../
+
+# install openjdk.
+apt-get install openjdk-8-jdk -y
+apt-get -o Dpkg::Options::="--force-overwrite" install openjdk-9-jdk -y
+
+# install nginx. Remove the hash character below to do it if needed.
+# apt-get install nginx -y
```

Comment:
Why is this commented?

Reply:
I thought that was only needed to install nginx at the load balancer VM.

Reply:
That's correct. So please add a new script to setup load balancer vm instead of leaving it here.

Reply:
Working on it...

Reply:
... done. :)